libs/utils/lana.js

```diff
@@ -40,8 +40,8 @@
     });
   }
 
-  function log(msg, options) {
-    msg = msg && msg.stack ? msg.stack : msg;
+  function log(msg = "", options) {
+    msg = msg.stack ? msg.stack : msg;
```

Comment:
Lana has to be IE11 compatible so default parameters are not supported
```suggestion
  function log(msg, options) {
    if (!msg) msg = '';
    msg = msg.stack ? msg.stack : msg;
```